Foundation/URLSession/http/HTTPURLProtocol.swift

```diff
@@ -557,7 +561,9 @@ extension _HTTPURLProtocol: _EasyHandleDelegate {
             completeTask()
         case .failWithError(let errorCode):
             internalState = .transferFailed
-            failWith(errorCode: errorCode, request: request)
+            let error = NSError(domain: NSURLErrorDomain, code: errorCode,
+                                userInfo: [NSLocalizedDescriptionKey: "Completion failure"])
```

Comment:
We should be calling that new `NSLocalizedString` function here (once its signature is fixed).

Reply:
Signature fixed. NSLocalizedString() is called indirectly in the creation of the final URLError struct in failWith(error:….